### PR TITLE
Add 'Yhdistys'-page with membership form.

### PR DESCRIPTION
**Pull request by Keksike**
Frontend ready, backend still WIP.

Code could be refactored here and there... maybe later.

Maybe pend merging until backend ready too.

### CODE REVIEW

File: src/js/components/membershipInfoForm.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+      var inputClasses = classSet({
+        'input': true,
+        'has-error': _.includes(fieldsWithErrors, fieldName)
+      });
+
+      inputFields.push((
+        <input
+          key         = { fieldName }
+          className   = { inputClasses }
+          type        = 'text'
```
Comment (peritpatrio) [inline src/js/components/membershipInfoForm.js]: Emaililla olis hyvä olla `type="email"`?

File: src/js/components/stripeCheckout.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      return <p>Virhe maksaessa! Ota yhteyttä info@koodiklinikka.fi</p>
+    } else if (this.state.sending) {
+      return <img src="../images/ajax-loader.gif" alt="Odota hetki..." height="42" width="42"></img>
+    } else {
+      return (<StripeCheckout
+        amount      = { 1000 }
+        currency    = 'EUR'
+        description = 'Jäsenmaksu'
+        email       = { this.props.userInfo.email }
+        image       = 'https://avatars3.githubusercontent.com/u/10520119?v = 3&s = 200'
```
Comment (peritpatrio) [inline src/js/components/stripeCheckout.js]: Pitäiskö tässä olla `https://avatars3.githubusercontent.com/u/10520119?v=3&s=200`?

File: src/yhdistys.jade [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+                    Kuka tahansa Slack:iin rekisteröitynyt käyttäjä voi liittyä yhdistyksen jäseneksi maksamalla kertaluontoisen <b>10€</b> jäsenmaksun.
+                  p.
+                    Nimellisellä jäsenmaksulla tulemme tekemään tapahtumistamme vieläkin mielenkiintoisempia ja kattamaan kustannuksia jäsenmäärämme kasvaessa.
+                  p.
+                    Jos tarvitset Slack-kutsun, siirry <a target="_blank" href="/">etusivulle</a>.
+                  p.
+                    Jäsenrekisterin rekisteriseloste luettavissa <a target="_blank" href="https://docs.google.com/document/d/1w-NZkf1EFYMC8a4LsoZouhpMVKORmPi8UFWRhurr0oM/edit?usp=sharing">Google Drivessä</a>.
+
+
+      footer
```
Comment (rikukissa) [inline src/yhdistys.jade]: Could we have these bits in another jade file that we could import to both views?

Comment (rikukissa) [inline src/yhdistys.jade]: Maybe the <head> as well

**Pull request merged**